src/soplex.hpp

```diff
@@ -3860,6 +3860,114 @@ bool SoPlexBase<R>::getPrimalRational(VectorBase<Rational>& vector)
       return false;
 }
 
+template <class R>
+bool SoPlexBase<R>::getRowPrimalValue(int i, R& value) {
+  if(_realLP != 0 && hasSol())
+  {
+    _syncRealSolution();
+    value = _realLP->computePrimalActivity(i, _solReal._primal, true);
+    return true;
+  }
+  else
+    return false;
+}
+
+template <class R>
+bool SoPlexBase<R>::getRowPrimalValueRational(int i, Rational& value) {
+  if(_rationalLP != 0 && hasSol())
+  {
+    _syncRationalSolution();
+    value = _rationalLP->computePrimalActivity(i, _solRational._primal);
```

Comment:
The `unscaled` flag always defaults to `true`, right? So why do you set it sometimes, but not all the time?

Reply:
This confused me too, but there are a couple of instances when dealing with `_realLP` that true is used explicitly, while in most cases, it is not. I had no idea if this difference had any semantic meaning, so I kept it. 

E.g.
https://github.com/scipopt/soplex/blob/3982032d3aa38d37697b82ffd9c95cf225b772ce/src/soplex.hpp#L1267

If you say it doesn't, it would be best to remove it

Reply:
While I did not write this code myself, I think it is safe to remove it.